std/experimental/allocator/building_blocks/affix_allocator.d

```diff
@@ -69,11 +70,21 @@ struct AffixAllocator(Allocator, Prefix, Suffix = void)
     static if (stateSize!Allocator)
     {
         Allocator _parent;
```

Comment:
I think this, `Allocator _parent`, should be made private as we want the user to use the `parent()` method.